cookbook/case_studies/bioinformatics/in_container.mk

```diff
@@ -9,7 +9,7 @@ $(SERIALIZED_PB_OUTPUT_DIR): clean
 
 .PHONY: serialize
 serialize: $(SERIALIZED_PB_OUTPUT_DIR)
-	pyflyte --config /root/sandbox.config serialize workflows -f $(SERIALIZED_PB_OUTPUT_DIR)
+	pyflyte --pkgs bioinformatics.blast package -f
```

Comment:
looks like an image is needed here: https://github.com/flyteorg/flytesnacks/actions/runs/4184441639/jobs/7250791149#step:4:23109